services/src/main/java/com/linkedin/kmf/common/Utils.java

```diff
@@ -27,6 +30,9 @@
 public class Utils {
   private static final Logger LOG = LoggerFactory.getLogger(Utils.class);
 
+  private static final int ZK_CONNECTION_TIMEOUT = 30_000;
```

Comment:
Should this be 3000 instead of 30_000? Would ZK_CONNECTION_TIMEOUT_MS and ZK_SESSION_TIMEOUT_MS be better names?